lib/sshkit/configuration.rb

```diff
@@ -29,6 +29,10 @@ def default_runner
       @default_runner ||= :parallel
     end
 
+    def default_runner_config
+      @default_runner_config ||= {}
```

Comment:
@wcmonty Thanks very much for doing this. I hadn't really understood the complexity of what this would mean for the :in key in the config. I tried poking this around a bit, and I think this might be an improvement, although I might have missed something: https://github.com/robd/sshkit/commit/9c8714009c01fcf968511a366b27d4e50ac2482a